graphql/src/main/java/com/linecorp/armeria/server/graphql/DefaultGraphqlErrorHandler.java

```diff
@@ -0,0 +1,58 @@
+package com.linecorp.armeria.server.graphql;
+
+import static com.linecorp.armeria.server.graphql.DefaultGraphqlService.newExecutionResult;
+
+import javax.annotation.Nonnull;
+
+import com.linecorp.armeria.common.HttpResponse;
+import com.linecorp.armeria.common.HttpStatus;
+import com.linecorp.armeria.common.MediaType;
+import com.linecorp.armeria.common.annotation.Nullable;
+import com.linecorp.armeria.server.ServiceRequestContext;
+
+import graphql.ExecutionInput;
+import graphql.ExecutionResult;
+import graphql.validation.ValidationError;
+/*
+ * Copyright 2022 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+enum DefaultGraphqlErrorHandler implements GraphqlErrorHandler {
+    INSTANCE;
+
+    @Nonnull
+    @Override
+    public HttpResponse handle(
+            ServiceRequestContext ctx,
+            ExecutionInput input,
+            ExecutionResult result,
+            MediaType negotiatedProduceType,
+            @Nullable Throwable cause
+    ) {
```

Comment:
We don't put this parenthesis in a separate line. 😉 
```suggestion
            @Nullable Throwable cause) {
```